### ISSUE TITLE
Error on workspaces with spaces in path

### ISSUE THREAD

**Issue by JeanMertz** [sagiegurari/cargo-make#518]
I have been unable to figure out where this happens, but when using a workspace project with a space in the root path, `cargo make` fails to run:

```
[cargo-make] INFO - Running Task: workspace
/var/folders/xc/m8n0122s0r375wygd39_g2mc0000gn/T/jean/fsio/8gZXp4Vo7N.sh: line 2: cd: /Users/jean/My: No such file or directory
[cargo-make] ERROR - Error while executing command, exit code: 1
[cargo-make] WARN - Build Failed.
```

If I run with `--no-workspace`, it works as expected.

**Comment by sagiegurari** [comment]
oh i think i know what the issue is.... i'll probably have to fix it in run_script
https://github.com/sagiegurari/run_script/blob/master/src/runner.rs#L90
probably add " before and after.

**Comment by sagiegurari** [comment]
should be fine now. just reinstall cargo-make and tell me if its solve please.

**Comment by sagiegurari** [comment]
@JeanMertz can you please verify on your end?

**Comment by sagiegurari** [comment]
Fix is released in 0.32.13 version

**Comment by JeanMertz** [comment]
Thanks for the fix @sagiegurari 👍
